frontend/src/app/containers/App.js

```diff
@@ -70,7 +70,7 @@ class App extends Component {
     if (window.ga && ga.loaded) {
       const data = dataIn || {};
       data.hitType = type;
-      data.page = (pathname === '/') ? pathname : '/' + pathname;
+      data.location = window.location;
```

Comment:
Separately, the right place to make this change is probably in `measurePageview`, rather than here:

```js
const { pathname } = window.location;
```